nautobot/extras/templates/extras/configcontextschema.html

```diff
@@ -1,19 +1,11 @@
 {% extends 'generic/object_detail.html' %}
 {% load helpers %}
 
-{% block nav_tabs %}
-        <li role="presentation"{% if active_tab == 'configcontextschema' %} class="active"{% endif %}>
```

Comment:
Can you double-check that the tabs on this page work correctly? I see that `nautobot.extras.views.ConfigContextSchemaView` is rendering this page with `"active_tab": "configcontextschema"` - at the very least that might not be necessary any more?

Reply:
Yes, it now works correctly